### ISSUE TITLE
JSONDecodeError when 429 Error

### ISSUE THREAD

**Issue by myangelaku** [circleguard/circleguard#122]
Might have replays that this happens to to be retried after a bit and add a better error message

(This was very likely caused by me browsing the osu! page while using Circleguard)

```
[ERROR] 2021/01/07 08:24:24.0932 JSONDecodeError, response: <Response [429]>, response.text: '<html>\r\n<head><title>429 Too Many Requests</title></head>\r\n<body>\r\n<center><h1>429 Too Many Requests</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>\r\n' (ossapi.ossapi, ossapi.py:58)
Traceback (most recent call last):
  File "site-packages\ossapi\ossapi.py", line 56, in _process_url
  File "site-packages\requests\models.py", line 900, in json
  File "json\__init__.py", line 348, in loads
  File "json\decoder.py", line 337, in decode
  File "json\decoder.py", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by tybug** [comment]
I think that replays this happens to \*are* retried. The run should have completed even if this exception was thrown (we wait 3 seconds and then try again) - let me know if that didn't happen.

**Comment by tybug** [comment]
better error message fixed in 9e80b16261574a2c590a1ebe9e0152eee44b4685. Closing as I believe we already implement the retrying.
